protocol.py

```diff
@@ -310,15 +252,15 @@ def worker(self) -> None:
 
             # https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-createwindowexw
             hwnd = CreateWindowExW(
-                0,                       # dwExStyle
+                0,  # dwExStyle
                 wndclass.lpszClassName,  # lpClassName
-                "DDE Server",            # lpWindowName
-                0,                       # dwStyle
+                "DDE Server",  # lpWindowName
+                0,  # dwStyle
                 CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT,  # X, Y, nWidth, nHeight
                 self.master.winfo_id(),  # hWndParent # Don't use HWND_MESSAGE since the window won't get DDE broadcasts
-                None,                    # hMenu
-                wndclass.hInstance,      # hInstance
-                None                     # lpParam
+                None,  # hMenu
+                wndclass.hInstance,  # hInstance
+                None  # lpParam
```

Comment:
Do we want to remove the visual padding here?

Reply:
Think this was an automatic reformatting. Will undo - no we don't want to remove it here (for now).